### ISSUE TITLE
New release to include bindgen update for crates.io

### ISSUE THREAD

**Issue by floroeske** [dankamongmen/libnotcurses-sys#30]
Hi,

Was playing with this a bit and had some trouble getting the library to build. I ended up with some bindgen errors. I noticed the latest release on crates.io (libnotcurses-sys v3.10.0) doesn't include the bindgen version bump https://github.com/dankamongmen/libnotcurses-sys/commit/a474d54b66e30549d1f76d55843c49d32aa506f7.

Seems to work if you point cargo to git main.
```
[dependencies]
libnotcurses-sys = { git = "https://github.com/dankamongmen/libnotcurses-sys.git", branch = "main" }
# libnotcurses-sys = "=3.10.0"
```
Was just wondering if you could bump the version on creates.io to match main.

**Comment by dankamongmen** [comment]
not sure if @joseluis is watching this; if not, i can step in and do this. thanks for the report!

**Comment by joseluis** [comment]
Sure, I think I could also update bindgen from 0.62 to 0.66, and libc from 0.148 to 0.152 since those are the minimum common versions available in debian testing and fedora rawhide, and publish the new version.

**Comment by joseluis** [comment]
published v3.10.1
